src/modules/Host.rb

```diff
@@ -349,9 +376,7 @@ def SetModified
       nil
     end
 
-    publish :variable => :hosts, :type => "map <string, list <string>>"
     publish :variable => :modified, :type => "boolean"
-    publish :variable => :write_only, :type => "boolean"
```

Comment:
I have double checked that these variables are indeed not used in other packages.